pallets/manual-migrations/src/lib.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019-2023 PureStake Inc.
```

Comment:
```suggestion
// Copyright 2024 Moonbeam foundation
```

New files should be licensed under the Moonbeam Foundation